### PR TITLE
Popular is popular

### PR DESCRIPTION
**Pull request by keithblaha**
:eyeglasses: @prashtx @schwers @nramadas 

looking for a quick review, would like to deploy today if possible

see https://reddit.atlassian.net/browse/HOT-117 for more details

### CODE REVIEW

**Comment by prashtx** [comment]
:fish: assuming the build/tests are successful

File: src/app/less/icon-font.less [hunk 1]
```diff
@@ -166,6 +166,23 @@ button, a {
...
+  // custom for use in CommunityOverlayMenu
+  vertical-align: middle;
+  position: relative;
+  top: -2px;
+
+  line-height: 39px;
+  padding-left: 1px;
+  font-size: 21px;
+
+  background-color:@mint;
```
Comment (schwers) [inline src/app/less/icon-font.less]: nitpick: missing a space before `@mint`

Comment (keithblaha) [inline src/app/less/icon-font.less]: ah good eye- ty

**Comment by keithblaha** [comment]
thank you for the quick review!

**Pull request merged**